src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -25,8 +25,52 @@
 
 _THIS_DIR = os.path.dirname(__file__)
 _OPTIONS_FILE = os.path.join(_THIS_DIR, "fluent_launcher_options.json")
-FLUENT_VERSION = "22.2"
-PIM_FLUENT_PRODUCT_VERSION = FLUENT_VERSION.replace(".", "")
+FLUENT_VERSION = ["22.2"]
+PIM_FLUENT_PRODUCT_VERSION = FLUENT_VERSION[0].replace(".", "")
+FLUENT_EXE_PATH = []
+
+
+class FluentVersion(Enum):
+    """Contains the standard ansys / fluent release."""
+
+    version_22R2 = "22.2"
+    version_23R1 = "23.1"
+
+    @staticmethod
+    def get_version(version: str) -> "FluentVersion":
+        """Returns the available versions based on the version in string
+        format."""
+        for v in FluentVersion:
+            if version == v.value:
+                return v
+        else:
+            raise RuntimeError(f"The passed version '{version}' does not exist.")
+
+
+def set_fluent_path(fluent_exe_path: Union[str, Path]) -> None:
+    """Lets the user set the fluent installation path manually.
+
+    This supersedes the fluent path set in the environment variable
+    """
+    if str(fluent_exe_path).endswith("fluent.exe"):
+        FLUENT_EXE_PATH.append(str(fluent_exe_path))
+    else:
+        raise RuntimeError(
+            f"The passed path '{fluent_exe_path}' does not contain a valid fluent executable file."
+        )
+
+
+def set_ansys_version(version: Union[str, float, FluentVersion]) -> None:
+    """Lets the user set the fluent version manually.
+
+    Only works if the provided ansys version is installed and the
+    environment variables are updated properly. This supersedes the
+    fluent path set in the environment variable
+    """
+    if type(version) in [float, str]:
+        version = FluentVersion.get_version(str(version))
+    if version in FluentVersion or str(version) in FluentVersion.value:
+        FLUENT_VERSION[0] = version.value
```

Comment:
@prmukherj PIM_FLUENT_PRODUCT_VERSION depends on FLUENT_VERSION. So we should update it too.

Reply:
ok, done. Thank you